assets/javascripts/kitten/components/form/checkbox.test.js

```diff
@@ -34,4 +34,18 @@ describe('Checkbox with default props', () => {
       expect(input).to.have.attr('id', 'input-1')
     })
   })
+
+  describe('renders children', () => {
```

Comment:
Ce qu'on veux tester c'est "Sur une Checkbox si on passe des children alors on render ses children". Ici les `describe` doivent être correspondre au "sur" et au "si" et les `it` au "alors".

En anglais si on ajoute tous les describe à la suite actuellement ça ferait la phrase  :
- `Checkbox with default props` `<Checkbox />` `renders children` it `it children`

Ça serait sans doute mieux :
- `<Checkbox />` `with children` it `renders the children inside the label`

Du coup je ferais plutôt `describe('with children',` et plus bas `it('renders the children inside the label'`.

Tu peux voir la phrase quand tu lance les specs :

<img width="690" alt="screen shot 2017-01-24 at 11 16 00" src="https://cloud.githubusercontent.com/assets/132/22243186/88b77824-e226-11e6-951f-c58dfe068c3b.png">